question_one.py

```diff
@@ -0,0 +1,45 @@
+numbers = [2, 4, 5, 2, 12, 44, 5, 1, 2, 3]
+
+print('\nA. Length of the list \nB. Display first 3 numbers \nC. Display sum of odd numbers \nD. Number of duplicate numbers \nE. Display list without duplicate numbers\n\nSelect Option (A,B,C,D,E) to Continue.....')
+
+option=input().capitalize()
+    
+class UserFunction:
```

Comment:
Actually the naming convention is wrong here.

You have defined a class but you are calling it `UserFunction` function? Why?